xql/src/xql/apply.py

```diff
@@ -432,28 +432,19 @@ def display_table_dataset_map(cmd: str) -> None:
         list_key_values(table_dataset_map)
 
 
-def display_result(result: t.Any, query: str) -> None:
-    """
-    Display the result of a query.
-
-    Args:
-        result (Any): The result to be displayed.
-    """
-    if isinstance(result, xr.Dataset):
-        df = filter_records(convert_to_dataframe(result), query)
-        print(df)
-    else:
-        print(result)
-
 @timing
-def run_query(query: str) -> None:
+def run_query(query: str, connect_cluster = True) -> None:
```

Comment:
what is the use of 'connect_cluster' argument here ?

Reply:
Removed.